.env.development

```diff
@@ -13,7 +13,8 @@ REACT_APP_S3_BUCKET=static-dev.kolable.com
 
 # Server
 REACT_APP_KOLABLE_SERVER_ENDPOINT=https://demomo.kolable.dev/enterprise
-REACT_APP_LODESTAR_SERVER_ENDPOINT=https://demomo.kolable.dev/api/v2
+# REACT_APP_LODESTAR_SERVER_ENDPOINT=https://demomo.kolable.dev/api/v2
+REACT_APP_LODESTAR_SERVER_ENDPOINT=http://localhost:8081/api/v2
```

Comment:
Be careful not to include tests in the commit

Reply:
Ooooooh!! Thank you for checking~